src/main/java/com/auth0/json/mgmt/Permission.java

```diff
@@ -12,96 +14,109 @@
 @JsonInclude(JsonInclude.Include.NON_NULL)
 public class Permission {
 
-  @JsonProperty("permission_name")
```

Comment:
whitespace changes, previous version was not properly formatted. Ignore up until the bottom where a new method is added for "getting the sources"